### DISCUSSION TITLE
Mention member pada group

### DISCUSSION THREAD

**Discussion by melodiazz** [kirimwa-id/forum#41] [Tanya Jawab] [answered]
hi pak @rioastamal 

Apakah possible untuk melakukan mention terhadap member yang ada di group wa? misal menggunakan @62XXXXXXX. Thanks

**Comment by rioastamal** [comment]
Halo @melodiazz, saat ini masih belum bisa. Sudah saya masukkan roadmap #43. Nanti bisa ditrack disana.

**Comment by rioastamal** [comment, marked as answer]
@melodiazz Fitur ini telah hadir di versi 1.0-RC8. Lihat rilis lengkapnya di https://github.com/kirimwa-id/forum/discussions/50
